plugins/rebalance/rest.go

```diff
@@ -0,0 +1,279 @@
+package rebalance
+
+import (
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gorilla/mux"
+	"github.com/pborman/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+// RebalanceStatus represents Rebalance Status
+type RebalanceStatus uint64
```

Comment:
Should this be duplicated across files?

Reply:
Answering here since it is generic golang question. Scope of this definition is that directory. All files of a directory belongs to same package in Go, So this will be available for all files